kubernetes/samples/scripts/create-weblogic-domain/model-in-image/domain-resources/JRF/mii-initial-d1-JRF-v1.yaml

```diff
@@ -92,6 +92,21 @@ spec:
   clusters:
```

Comment:
This file and the other domain resource files in the MII sample are generated from a template and checked-in.  
So the corresponding file 'mii-domain.yaml.template-JRF' and also the WLS template file 'mii-domain.yaml.template-WLS' also need to be modified.   

If you want to test that templates and sample files match, run 'cd src/integration-tests/model-in-image/mii-sample-wrapper  ; ./generate-sample-doc.sh '.  This will generate YAML from the templates and compare it to the files checked into source.  It should pass.   (The same script is called as part of the MII sample testing - which isn't merged into develop yet.)

Reply:
I ran this on my Linux box and it succeeded; however, it's not clear that it compared what is currently in the tree -- where would I see that?  It ends like this:
```
@@
@@ Info: Finished 'generate-sample-doc.sh'! See target directory '/tmp/mii-sample'.
@@
```